cloud-aws/src/main/resources/templates/aws-cf-dbstack.ftl

```diff
@@ -87,7 +101,35 @@
   },
 
   "Resources" : {
-
+  <#if !hasSecurityGroup>
+        "VPCSecurityGroup": {
+          "Type" : "AWS::EC2::SecurityGroup",
+          "Properties" : {
+              "GroupDescription" : "Security group attached to the database server",
+              "GroupName" : { "Ref":"DBSecurityGroupNameParameter"},
+              "SecurityGroupEgress" : [
+                    {
+                      "CidrIp": "0.0.0.0/0",
+                      "IpProtocol": "-1"
+                    }
+              ],
+              "SecurityGroupIngress" : [
+                    {
+                      "IpProtocol" : "tcp",
+                      "FromPort": 5432,
+                      "ToPort" : 5432,
```

Comment:
Are we always setting up the DB to listen on this port? If the port is configurable elsewhere, we should respect that configuration here.

Reply:
That's a great point. Port can be specified from the outside, I modify the template. Thanks!